src/commands/createHttpFunction.ts

```diff
@@ -65,3 +62,29 @@ function generateSuffixedName(preferredName: string, i: number): string {
 async function isNameAvailable(folderPath: string, newName: string): Promise<boolean> {
     return !(await fse.pathExists(path.join(folderPath, newName)));
 }
+
+async function getFunctionsApi(context: IActionContext): Promise<AzureFunctionsExtensionApi> {
+    const funcExtensionId: string = 'ms-azuretools.vscode-azurefunctions';
+    const funcExtension: vscode.Extension<AzureExtensionApiProvider> | undefined = vscode.extensions.getExtension(funcExtensionId);
+
+    if (funcExtension) {
+        if (!funcExtension.isActive) {
+            await funcExtension.activate();
+        }
+
+        // The Functions DB extension just recently added support for 'AzureExtensionApiProvider' so we should do an additional check just to makes sure it's defined
+        // tslint:disable-next-line: strict-boolean-expressions
+        if (funcExtension.exports) {
+            return funcExtension.exports.getApi<AzureFunctionsExtensionApi>('^1.0.0');
```

Comment:
You need to keep the version at `^1.1.0`

`createFunction` wasn't added until 1.1.0: https://github.com/microsoft/vscode-azurefunctions/pull/2077/files#diff-45327f86d4438556066de133327f4ca2R87